.github/workflows/composites/load-docker-images/action.yaml

```diff
@@ -0,0 +1,27 @@
+name: install docker images
+description: load docker images from tar
+runs:
+  using: "composite"
+  steps:
+    - name: load docker images
+      shell: bash
+      run: |
+        docker load -i /tmp/docker/images/spring-cloud-kubernetes-client-config-it.tar
```

Comment:
this looks scary and I thought initially a a simple fact: why load every image in all steps, if I know that some steps either do not requires any images at all (there are no integration tests in this index) or they might need 1 or 2 images at all. But it takes only 1 minute to load, so far too little to invest any additional time here, imho

Reply:
What about here?  If I add a new integration tests wouldnt I need to add the docker image here?

Reply:
OMG!!! I am an idiot, really sorry. This was never intended to load images like this, but instead iterate and find proper names. This gain proves my theory that you have an eye for where I am minor issues in my PRs. I love it!